task/bq2bq/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/googleapis/google-cloud-go-testing v0.0.0-20210719221736-1c9a4c676720
 	github.com/hashicorp/go-hclog v1.2.0
 	github.com/mitchellh/hashstructure/v2 v2.0.2
-	github.com/odpf/optimus v0.2.2-0.20220606050552-243f0c02d055
+	github.com/odpf/optimus v0.2.6-rc.1.0.20220726091918-0b5282570d77
```

Comment:
same here

Reply:
updated. but, note that we are using the latest (at the time of writing) commit of the [window branch](https://github.com/odpf/optimus/tree/window-versioning).